### PR TITLE
Add taxprofiler to pipeline options

### PR DESCRIPTION
**Pull request by sofstam**
### This PR adds | fixes:
-  Adds taxprofiler to pipeline options

**How to prepare for test**:
- [ ] `ssh` to ...
- [ ] Install on stage:
`bash servers/resources/SERVER.scilifelab.se/update-[THIS_TOOL]-stage.sh [THIS-BRANCH-NAME]`

### How to test:
-

### Expected outcome:
- [ ]

### Review:
- [ ] Code approved by
- [ ] Tests executed by
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ Try to use the following format:
 ### Fixed
 -->
 
+## [0.12.0]
+
```
Comment (fevac) [inline CHANGELOG.md]:
```suggestion

```

Remove empty line

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,10 @@ Try to use the following format:
...
+### Added [cg]
+- Adds taxprofiler to pipeline options
```
Comment (fevac) [inline CHANGELOG.md:21]:
```suggestion
- Adds taxprofiler to pipeline options

```

Add line

File: cgmodels/cg/constants.py [hunk 3]
```diff
@@ -21,3 +21,4 @@ class Pipeline(StrEnum):
     RSYNC: str = "rsync"
     SARS_COV_2: str = "sars-cov-2"
     SPRING: str = "spring"
+    TAXPROFILER: str = "taxprofiler"
```
Comment (fevac) [inline cgmodels/cg/constants.py:24]: 🤩

**Review by fevac — CHANGES_REQUESTED** [review]
Awesome!! 🌟 🌟 Exciting to see taxprofiler coming along!

**Review by karlnyr — APPROVED** [review]
👍

**Pull request merged**